services/spar/src/Spar/Intra/Brig.hs

```diff
@@ -102,11 +101,16 @@ createBrigUser suid (Id buid) teamid mbName managedBy = do
   uname :: Name <- case mbName of
     Just n -> pure n
     Nothing -> do
+      -- 1. use 'SAML.unsafeShowNameID' to get a 'Name'.  rationale: it does not need to be
+      --    unique.
+      -- 2. do not throw an error if it is too long, but truncate it.  rationale: it's an
```

Comment:
User names are strictly cosmetic right? Like you don't use them as identifiers to log in or anything right?

I once had my ISP silently strip characters from my password when I updated it (they removed special characters); so I would set my password and then it didn't work, and that was a terrible experience so as long as that can't happen here then I'm fine with it.

Reply:
ok, since you are the second to be vaguely worried about silently stripping characters here, i will change it.  i will also change the magic number to a well-named identifier, but that's unrelated, so i will do it in a separate PR.

more commits coming up!